.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: python
-python:
-    - 2.7
-    - 3.5
-    - 3.6
-    - 3.7
-    - 3.8
-    - pypy
-    - pypy3
-env:
-  global:
-    - ZOPE_INTERFACE_STRICT_IRO: 1
```

Comment:
We had this in CI, and I thought it was fine not to have it in the github workflow file, because we now use tox, and surely tox will have the setenv.

And it did, but you're removing it.

Reply:
Without a `passenv` declaration in `tox.ini` this does nothing because `tox` ignores it. I added `setenv` in https://github.com/zopefoundation/zope.publisher/pull/58/commits/a897f45ee32214bc09d2d1bdedb75edb97d67de5 which I think should be fine.